specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp

```diff
@@ -199,6 +199,15 @@ model ChatChoice {
   @projectedName("json", "delta")
   @projectedName("csharp", "InternalStreamingDeltaMessage")
   delta?: ChatMessage;
+
+  @doc("""
+    Information about the content filtering category (hate, sexual, violence, self_harm), if it 
+    has been detected, as well as the severity level (very_low, low, medium, high-scale that 
+    determines the intensity and risk level of harmful content) and if it has been filtered or not.
+    """)
+  @added(ServiceApiVersions.v2023_06_01_Preview)
+  @projectedName("json", "content_filter_results")
+  contentFilterResults?: ContentFilterResults;
```

Comment:
Does this property need to be added to the Choice model just like it is being added here to the ChatChoice model?

Reply:
Egads, it does indeed! Update just pushed.